### DISCUSSION TITLE
In Opencart's v3.x default checkout "Payment method required!" and the checkout can not complete

### DISCUSSION THREAD

**Discussion by kanenas** [opencart/opencart#14018] [Q&A]
We are using **Opencart v3.0.3.8** with the default checkout and we get the `$this->language->get('error_payment');` and the checkout can not complete asking for "**_Payment method required!_**".
In `catalog/controller/checkout/payment_method.php` for some reason `$this->session->data['payment_methods']` is empty just before `save()`.

Does anyone else has experienced this?

**Comment by mhcwebdesign** [comment]
This is most likely an issue with a wrongly configured or missing payment method. Please repost on the [OpenCart forum](https://forum.opencart.com/viewforum.php?f=202) with some more details to get some help.

**Reply by kanenas** [reply]
@mhcwebdesign thank you for your answer. Did you notice that I am talking about `payment_methods` and not `payment_method`?
